sdk/js-proto-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@certusone/wormhole-sdk-proto-node",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
Is there a .MD file that needs to be updated for release notes?

Reply:
Not currently for the proto packages, but we could consider adding them in the future